templates/base.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{ title }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Genearl Stylesheets -->
+    <link rel="stylesheet" href="/static/styles/eventslyStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/navbarStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/footerStyleSheet.css">
+
+    <!-- Specific Styles NOTE: (variable shows as error, but it is not)-->
+    <style type="text/css">{{ pageStyles }}</style>
+
+    <!-- Eventsly Scripts -->
+    <script src='/static/scripts/navScripts.js' async></script>
+
+    <!-- Font Awesome JS load for Eventsly Logo -->
+    <script src="https://kit.fontawesome.com/3f6d9190cb.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font API load for Eventsly Logo -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
+
+    <!-- block for page-specific head content (stylesheets, scripts, etc) -->
+    {% block head %} {% endblock %}
```

Comment:
Notice that `{% block head %} {% endblock %}` are on each .html page. This allows us to specify page-specific code on each .html page within in the <head> tags and after the general code.